src/scratch/UCERF3/inversion/InversionTargetMFDs.java

```diff
@@ -135,11 +138,16 @@
 	
 	public final static double FAULT_BUFFER = 12d;	// buffer for fault polygons
 
-
+	/**
+	 * Implicit constructor required for subclassing
+	 */
+	public InversionTargetMFDs() {
```

Comment:
Can you set this to protected? Don't want people randomly instantiating this object without doing the setup.

Reply:
done, thanks!